packages/paste-website/src/pages/patterns/status/index.mdx

```diff
@@ -400,7 +400,7 @@ to ensure that the icon and the text are vertically aligned.
 
 ### Status badges
 
-For convience and consistency, we have created a `StatusBadge` (coming soon) component that can be used to compose a status icon with a text label.
+For convience and consistency, we have created a [`StatusBadge`](/components/status-badge) component that can be used to compose a status icon with a text label.
 
 <LivePreview scope={{Box, StatusBadge}} language="jsx">
   {`<Box display="flex" flexDirection="column" rowGap="space40">
```

Comment:
todo: Replace the Status Badges in the sections below with just icon + text. These shouldn't use a badge since the border + color + font change adds unnecessary noise when looking at a lot of items together in a table layout:

- Positioning in a table
- Pairing process statuses with plain text statuses

Reply:
If it's not a blocker I'm going to skip this

Reply:
sure, i'll ticket it for later